### DISCUSSION TITLE
Can we show the name of the multifield item when we click on each multi-field item

### DISCUSSION THREAD

**Discussion by snallapa8** [merkle-open/aem-generic-multifield#67] [General]
Like example component as the name of the dialog - when we click on pencil - Can we make the name of the multi field to show as dialog name?

**Comment by michaelhoerenberg** [comment]
Hello @snallapa8,
just to be sure that i understood correctly what you are asking.

You want the jcr:title of the item dialog to be generic to a property value?
Can you maybe elaborate a bit or explain with faked screenshots?

**Comment by michaelhoerenberg** [comment]
Hello @snallapa8,
any further updates on this question?
